plugins/org.eclipse.glsp.server/src/org/eclipse/glsp/server/gmodel/GModelDiagramModule.java

```diff
@@ -49,20 +56,20 @@ protected Class<? extends GModelFactory> bindGModelFactory() {
    @Override
    protected void configureActionHandlers(final MultiBinding<ActionHandler> binding) {
       super.configureActionHandlers(binding);
-      binding.add(RequestClipboardDataActionHandler.class);
+      binding.add(GModelRequestClipboardDataActionHandler.class);
    }
 
    @Override
    protected void configureOperationHandlers(final MultiBinding<OperationHandler> binding) {
       super.configureOperationHandlers(binding);
-      binding.add(ApplyLabelEditOperationHandler.class);
-      binding.add(ChangeBoundsOperationHandler.class);
-      binding.add(ChangeRoutingPointsHandler.class);
+      binding.add(GModelApplyLabelEditOperationHandler.class);
+      binding.add(GModelChangeBoundsOperationHandler.class);
+      binding.add(GModelChangeRoutingPointsHandler.class);
+      binding.add(GModelDeleteOperationHandler.class);
+      binding.add(GModelReconnectEdgeOperationHandler.class);
+      binding.add(GModelPasteOperationHandler.class);
       binding.add(CutOperationHandler.class);
-      binding.add(DeleteOperationHandler.class);
       binding.add(LayoutOperationHandler.class);
```

Comment:
With the `LayoutOperationHandler` being generic and reusable for all implementations shouldn't it be bound as part of the parent module?

Same is probably true for the `CutOperationHandler`

Reply:
in fact, `LayoutOperationHandler` was already bound in the superclass anyway. But I now also pushed the cut operation handler up into the super class (and removed both from the subclass).